edit pass: Networking virtual datacenter

TrishaMc opened the pull request:
Edit complete. Revisions in progress.

VSC-Service-Account commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/vdc/networking-virtual-datacenter.md](https://github.com/TrishaMc/architecture-center/blob/networking-virtual-datacenter/docs/vdc/networking-virtual-datacenter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/vdc/networking-virtual-datacenter?branch=pr-en-us-856) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C16%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201810161510504401-856%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=cBCEeVBgObzHCVBEF7UoodsQ7HoiNgDuvX2Pooc2lSU%3D&st=2018-10-16T15%3A07%3A07Z&se=2018-11-16T15%3A12%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

VSC-Service-Account commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/vdc/networking-virtual-datacenter.md](https://github.com/TrishaMc/architecture-center/blob/networking-virtual-datacenter/docs/vdc/networking-virtual-datacenter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/vdc/networking-virtual-datacenter?branch=pr-en-us-856) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C16%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201810162349596582-856%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=YmUP8iGcpqP2Qu1WbDGy7xzGb4KZnQtiOPNiTGpFD%2F0%3D&st=2018-10-16T23%3A46%3A16Z&se=2018-11-16T23%3A51%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

VSC-Service-Account commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/vdc/networking-virtual-datacenter.md](https://github.com/TrishaMc/architecture-center/blob/networking-virtual-datacenter/docs/vdc/networking-virtual-datacenter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/vdc/networking-virtual-datacenter?branch=pr-en-us-856) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C17%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201810171846581083-856%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=apPCwKb47zNKOCi12Q5Z0S6Ck%2FkfD6OYJ4AyLL9X6MU%3D&st=2018-10-17T18%3A43%3A19Z&se=2018-11-17T18%3A48%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

VSC-Service-Account commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/vdc/networking-virtual-datacenter.md](https://github.com/TrishaMc/architecture-center/blob/networking-virtual-datacenter/docs/vdc/networking-virtual-datacenter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/vdc/networking-virtual-datacenter?branch=pr-en-us-856) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C17%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201810172350493120-856%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=aq2K9TkMITiuYsGRtW0YF4DKAX9Y3cLEC8fCRdAjvVk%3D&st=2018-10-17T23%3A47%3A04Z&se=2018-11-17T23%3A52%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

VSC-Service-Account commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/vdc/networking-virtual-datacenter.md](https://github.com/TrishaMc/architecture-center/blob/networking-virtual-datacenter/docs/vdc/networking-virtual-datacenter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/vdc/networking-virtual-datacenter?branch=pr-en-us-856) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C18%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201810181717056954-856%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=jo8IjzOWt1%2Fi9tFeLSaM6e9RghTIwENboARyI8IcDuo%3D&st=2018-10-18T17%3A13%3A25Z&se=2018-11-18T17%3A18%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

TrishaMc commented:
Hi @jonor, @MikeWasson, @Petertaylor9999, and @BrianBlanchard:

Please review the copy edit to this article, address the following comments, and let me know if I’ve changed the technical meaning anywhere. This is for VSTS work request [1354137](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1354137).

Thanks,
Trisha

Comments:

Lines 2 and 13: The article says “virtual datacenter” isn’t an Azure product “but the combination of various features and capabilities to meet your exact requirements.” Per the cloud style guide, only products are capitalized, not features. If this is a feature, the cloud style guide says not to use abbreviations for products or features. I’ve left “VDC,” but does that need to be spelled out? 
Also, “datacenter” is one word per the cloud style guide. 
Line 43: DevOps - From the Cloud style guide: “Precede names with Azure unless there are space constraints, for example, in the UI. Without Azure, the names are too generic and create potential legal problems with similar names outside Microsoft.” This sounds like “DevOps” shouldn’t be used as a separate term from “Azure DevOps.”
Line 47: vNet – Cloud style guide: “Don't use Vnet or VNET as an abbreviation. In technical documentation, you can use VNet as part of an example name for a virtual network created by the Virtual Network service. For example, MyVNet1, MyVNet2.”
Line 49: See Cloud style guide: https://worldready.cloudapp.net/Search/Index 
Line 68: “Azure account” Does this mean “Microsoft account”? https://worldready.cloudapp.net/Search/Index 
Line 76: LOB – From the Microsoft style guide: “line of business, line-of-business - Lowercase. Don’t hyphenate as a noun. Always hyphenate as an adjective. Don’t abbreviate as LOB.” 
Line 96: Azure site-to-site VPN - Not found as an Azure product or term in any style guides, so it should be lowercase.
Line 101: Does “access control lists” refer to Azure Access Control? It will be retired on November 7, 2018.
Line 161: I don’t find the role “Network Reader” in the cited article https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#classic-virtual-machine-contributor 
Line 165: Dev & test - I couldn’t find this term in style guides or online. I found “dev-test” on this Azure web page: https://azure.microsoft.com/en-us/solutions/dev-test/ 
Line 194: Should “Windows Firewall” be “Azure Firewall” or “Windows Defender Firewall”?
Line 210: “web application firewall” - lowercase features per Cloud style guide.
Line 219: If “vWAN” refers to the Azure product, it should be “Virtual WAN.”
Line 258: Is “Azure Activity Log” an Azure product? I didn’t find it in the style guides or cloud service directory, but it was in the linked article.
Line 260: “Azure Monitor diagnostic logs” is per the linked article.
Line 275: This link doesn’t work: [Azure Log Analytics][.../log-analytics/log-analytics-overview.md] Is the link I changed this to correct? https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-queries 
Line 302: “Web Apps feature of Microsoft Azure App Service” is per the Cloud style guide. It also says, “Don't refer to this feature as Azure Web Apps.”
Line 362: “Operations Management Suite [OMS]” is being rebranded to “Log Analytics.” Does it need to be revised in this article?

TrishaMc commented:
#sign-off

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+The VDC needs connectivity with external networks to offer services to customers, partners, and internal users. It usually needs connectivity not only to the internet but also to on-premises networks and datacenters.
 
-Customers can build their security policies to control what and how specific VDC hosted services are accessible to/from the Internet using [Azure Firewall][AzFW] or Network Virtual Appliances, and custom routing policies and network filtering ([User Defined Routing][UDR] and [Network Security Groups][NSG]). It's recommended that all Internet facing resources be additionally protected by [**Azure  ...
+Customers can build their security policies to control what and how specific VDC-hosted services are accessible to or from the internet with [Azure Firewall][AzFW] or network virtual appliances, custom routing policies, and network filtering. See information on [user-defined routes][UDR] and [network security groups][NSG]. We recommend that all internet-facing resources be additionally protected  ...
```
This paragraph is kind of a mess. It should be something like "Customers  control which services have access to and are accessible from the public internet using [Azure Firewall][AzFW] or other types of virtual network appliances (NVAs), custom routing policies using [user-defined routes][UDR], and network filtering using [network security groups][NSG]. We recommend that all internet-facing resources also be protected by the [Azure DDoS Protection Standard][DDOS]."

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+Customers can build their security policies to control what and how specific VDC-hosted services are accessible to or from the internet with [Azure Firewall][AzFW] or network virtual appliances, custom routing policies, and network filtering. See information on [user-defined routes][UDR] and [network security groups][NSG]. We recommend that all internet-facing resources be additionally protected  ...
 
-Enterprises often need to connect VDCs to on-premises data centers or other resources. The connectivity between Azure and on-premises networks is therefore a crucial aspect when designing an effective architecture. Enterprises have two different ways to create an interconnection between VDC and on-premises in Azure: transit over the Internet and/or by private direct connections.
+Enterprises often need to connect VDCs to on-premises datacenters or other resources. So the connectivity between Azure and on-premises networks is a crucial aspect when designing an effective architecture. Enterprises have two different ways to create interconnections between VDCs and on-premises in Azure: transit over the internet or private direct connections.
```
"connect VDCs to..." should be "connect their VDC implementations to..."

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+Enterprises often need to connect VDCs to on-premises datacenters or other resources. So the connectivity between Azure and on-premises networks is a crucial aspect when designing an effective architecture. Enterprises have two different ways to create interconnections between VDCs and on-premises in Azure: transit over the internet or private direct connections.
 
-An [**Azure Site-to-Site VPN**][VPN] is an interconnection service over the Internet between on-premises networks and the VDC, established through secure encrypted connections (IPsec/IKE tunnels). Azure Site-to-Site connection is flexible, quick to create, and does not require any further procurement, as all connections connect over the internet.
+An [Azure site-to-site VPN][VPN] is an interconnection service over the internet between on-premises networks and the VDC. It's established through secure encrypted connections, IPsec/IKE tunnels. An Azure site-to-site connection is flexible and quick to create. This VPN doesn't require any further procurement as all connections connect over the internet.
```
Should be "An [Azure site-to-site VPN][VPN] connects an Enterprise's on-premises network to their VDC implementation over the public internet. Traffic over the VPN is encrypted using IPSec and IKE tunneling."

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+For large numbers of VPN connections, [Azure Virtual WAN][vWAN] is a networking service that provides optimized and automated branch-to-branch connectivity through Azure. With Virtual WAN, you can connect and configure branch devices to communicate with Azure. This connection can be done either manually or by using preferred provider devices through a Virtual WAN partner. Preferred provider devic ...
 
-[**ExpressRoute**][ExR] is an Azure connectivity service that lets you create private connections between VDC and the on-premises networks. ExpressRoute connections do not go over the public Internet, and offer higher security, reliability, and higher speeds (up to 10 Gbps) along with consistent latency. ExpressRoute is very useful for VDCs, as ExpressRoute customers can get the benefits of compl ...
+With [ExpressRoute][ExR], an Azure connectivity service, you can create private connections between the VDC and on-premises networks. ExpressRoute connections don't go over the public internet. They offer higher security, reliability, and higher speeds, up to 10 Gbps, along with consistent latency. ExpressRoute is useful for VDCs as ExpressRoute customers get the benefits of compliance rules asso ...
```
"With [ExpressRoute][ExR], an Azure connectivity service, you can create private connections between the VDC and on-premises networks" should be "[ExpressRoute][ExR], an Azure connectivity service, enables private connections between the Enterprise's on-premises network and their VDC implementation."

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
 
 ### Topology
-_Hub and spokes_ is a model for extending a Virtual Data Center within a single Azure region.
+**Hub and spokes** is a model for extending a virtual datacenter within a single Azure region.
```
"Hub and spokes" is actually known as "hub and spoke".

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
 [![1]][1]
 
-The hub is the central zone that controls and inspects ingress and/or egress traffic between different zones: Internet, on-premises, and the spokes. The hub and spoke topology gives the IT department an effective way to enforce security policies in a central location, while reducing the potential for misconfiguration and exposure.
+The hub is the central zone that controls and inspects ingress or egress traffic between different zones: internet, on-premises, and the spokes. The hub and spoke topology gives the IT department an effective way to enforce security policies in a central location. It also reduces the potential for misconfiguration and exposure.
```
"The hub is the central zone..." should be "A hub is the central network zone..."

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+-   The Windows Active Directory infrastructure, required for user authentication of third parties that access from untrusted networks before they get access to the workloads in the spoke. It includes the related Active Directory Federation Services (AD FS).
+-   A DNS service to resolve naming for the workload in the spokes, to access resources on-premises and on the internet if [Azure DNS][DNS] isn't used.
+-   A public key infrastructure (PKI), to implement single sign-on on workloads.
+-   Flow control (TCP, UDP) between the spokes and internet.
```
Should be "Flow control of TCP and UDP traffic between the spoke network zones and the internet."

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+In Azure, every component, whatever the type, is deployed in an Azure subscription. The isolation of Azure components in different Azure subscriptions can satisfy the requirements of different lines of business. An example is setting up differentiated levels of access and authorization.
 
-A single VDC can scale up to large number of spokes, although, as with every IT system, there are platforms limits. The hub deployment is bound to a specific Azure subscription, which has restrictions and limits (for example, a max number of VNet peerings - see [Azure subscription and service limits, quotas, and constraints][Limits] for details). In cases where limits may be an issue, the archite ...
+A single VDC can scale up to a large number of spokes. But as with every IT system, there are platforms limits. The hub deployment is bound to a specific Azure subscription, which has restrictions and limits. An example is a maximum number of virtual network peerings. For details, see [Azure subscription and service limits, quotas, and constraints][Limits]. In cases where limits might be an issue ...
```
"A single VDC..." should be "A single VDC implementation..."

TrishaMc replied:
stopped work here - will continue tomorrow.

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
 [![3]][3]
 
-Spokes can also be interconnected to a spoke that acts as a hub. This approach creates a two-level hierarchy: the spoke in the higher level (level 0) become the hub of lower spokes (level 1) of the hierarchy. The spokes of VDC need to forward the traffic to the central hub to reach out either to the on-premises network or internet. An architecture with two levels of hub introduces complex routing ...
+Spokes can also be interconnected to a spoke that acts as a hub. This approach creates a two-level hierarchy: the spokes in the higher level, level 0, become the hub of lower spokes, level 1, of the hierarchy. The spokes of a VDC need to forward the traffic to the central hub to reach out either to the on-premises network or internet. An architecture with two levels of hub introduces complex rout ...
```
"The spokes of a VDC need to forward..." should be "The spokes of a VDC implementation are required to forward..."

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
 
 ### Components
-A Virtual Data Center is made up of four basic component types: **Infrastructure**, **Perimeter Networks**, **Workloads**, and **Monitoring**.
+A virtual datacenter is made up of four basic component types: **Infrastructure**, **Perimeter Networks**, **Workloads**, and **Monitoring**.
```
virtual datacenter implementation

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+A virtual datacenter is made up of four basic component types: **Infrastructure**, **Perimeter Networks**, **Workloads**, and **Monitoring**.
 
-Each component type consists of various Azure features and resources. Your VDC is made up of instances of multiple components types and multiple variations of the same component type. For instance, you may have many different, logically separated, workload instances that represent different applications. You use these different component types and instances to ultimately build the VDC.
+Each component type consists of various Azure features and resources. Your VDC is made up of instances of multiple component types and multiple variations of the same component type. For instance, you might have many different, logically separated workload instances that represent different applications. You use these different component types and instances to ultimately build the VDC.
```
"Your VDC..." should be "An Enterprise's VDC implementation..."

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
 The preceding high-level architecture of a VDC shows different component types used in different zones of the hub-spokes topology. The diagram shows infrastructure components in various parts of the architecture.
 
-As a good practice (for an on-premises DC or VDC) access rights and privileges should be group-based. Dealing with groups, instead of individual users helps maintaining access policies consistently across teams and aids in minimizing configuration errors. Assigning and removing users to and from appropriate groups helps keeping the privileges of a specific user up-to-date.
+As a good practice for an on-premises datacenter or VDC, access rights and privileges should be group based. Deal with groups instead of individual users to maintain access policies consistently across teams and minimize configuration errors. Assign and remove users to and from appropriate groups to keep the privileges of a specific user up to date.
```
"VDC" should be "a VDC implementation"

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+-   The operation and maintenance group, **CorpInfraOps** or **AppInfraOps,** has the responsibility of managing workloads in production. This group needs to be a subscription contributor on workloads in any production subscriptions. Some organizations might also evaluate if they need an additional escalation support team group with the role of subscription contributor in production and the centr ...
 
-A VDC is structured so that groups created for the central IT groups managing the hub have corresponding groups at the workload level. In addition to managing hub resources only the central IT groups would be able to control external access and top-level permissions on the subscription. However, workload groups would be able to control resources and permissions of their VNet independently on Cent ...
+A VDC is structured so that groups created for the central IT groups that manage the hub have corresponding groups at the workload level. In addition to managing hub resources, only the central IT groups would control external access and top-level permissions on the subscription. However, workload groups would control resources and permissions of their virtual network independently on the central ...
```
"A VDC..." should be "The VDC..." because we're talking about the concept here.

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
 [![6]][6]
 
-Infrastructure components provide an interconnection between the different components of a VDC, and are present in both the hub and the spokes. The responsibility for managing and maintaining the infrastructure components is typically assigned to the central IT and/or security team.
+Infrastructure components provide an interconnection between the different components of a VDC. They're present in both the hub and the spokes. The responsibility for managing and maintaining the infrastructure components is typically assigned to the central IT or security team.
```
"...a VDC" should be "...the VDC"

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+One of the primary tasks of the IT infrastructure team is to guarantee the consistency of IP address schemas across the enterprise. The private IP address space assigned to the VDC needs to be consistent. It can't overlap with private IP addresses assigned on your on-premises networks.
 
-While NAT on the on-premises edge routers or in Azure environments can avoid IP address conflicts, it adds complications to your infrastructure components. Simplicity of management is one of the key goals of VDC, so using NAT to handle IP concerns is not a recommended solution.
+NAT on the on-premises edge routers or in Azure environments can avoid IP address conflicts. But it adds complications to your infrastructure components. Simplicity of management is one of the key goals of a VDC. So using NAT to handle IP concerns isn't a solution that we recommend.
```
"...a VDC" should be "...the VDC"

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
--   [**RBAC**][RBAC]. Through RBAC, it is possible to map organizational role along with rights to access specific Azure resources, allowing you to restrict users to only a certain subset of actions. With RBAC, you can grant access by assigning the appropriate role to users, groups, and applications within the relevant scope. The scope of a role assignment can be an Azure subscription, a resource ...
--   [**VNet Peering**][VNetPeering]. The fundamental feature used to create the infrastructure of a VDC is VNet Peering, a mechanism that connects two virtual networks (VNets) in the same region through the Azure data center network, or using the Azure world-wide backbone across regions.
+-   [**Identity and directory services**][AAD]. Access to every resource type in Azure is controlled by an identity stored in a directory service. The directory service stores the users list and the access rights to resources in a specific Azure subscription. These services can exist in the cloud only. Or they can be synchronized with an on-premises identity stored in Azure Active Directory.
+-   [**Virtual network**][VPN]. Virtual networks are one of main components of a VDC. By using virtual networks, you can create a traffic isolation boundary on the Azure platform. A virtual network is composed of single or multiple virtual network segments. Each has a specific IP network prefix, a subnet. The virtual network defines an internal perimeter area where IaaS virtual machines and PaaS  ...
```
"...a VDC" should be "...the VDC"

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+-   [**Virtual network**][VPN]. Virtual networks are one of main components of a VDC. By using virtual networks, you can create a traffic isolation boundary on the Azure platform. A virtual network is composed of single or multiple virtual network segments. Each has a specific IP network prefix, a subnet. The virtual network defines an internal perimeter area where IaaS virtual machines and PaaS  ...
+-   [**UDR**][UDR]. Traffic in a virtual network is routed by default based on the system routing table. A user-defined route is a custom routing table that network administrators can associate to one or more subnets. This association overwrites the behavior of the system routing table and defines a communication path within a virtual network. The presence of UDRs guarantees that egress traffic f ...
+-   [**NSG**][NSG]. A network security group is a list of security rules that acts as traffic filtering on IP sources, IP destinations, protocols, IP source ports, and IP destination ports. The NSG can apply to a subnet, a virtual NIC card associated with an Azure VM, or both. NSGs are essential to implement a correct flow control in the hub and in the spokes. The level of security afforded by th ...
+-   [**DNS**][DNS]. The name resolution of resources in the virtual networks of a VDC is provided through the DNS. Azure provides DNS services for both [public][DNS] and [private][PrivateDNS] name resolution. Private zones provide name resolution both within a virtual network and across virtual networks. You can have private zones that span across virtual networks in the same region and also acro ...
```
"...a VDC" should be "...the VDC"

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
-In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall or a farm of firewalls and/or Web Application Firewalls (WAFs).
+[**Network virtual appliances**][NVA]. In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall instance or a farm of firewalls or web application firewall (WAF).
 
-Different LOBs commonly use many web applications, and these applications tend to suffer from various vulnerabilities and 
```
"...a VDC" should be "...the VDC"

Petertaylor9999 replied:
VDC should be VDC implementation

Petertaylor9999 replied:
VDC should be VDC implementation

Petertaylor9999 replied:
Last sentence "A VDC satisfies..." should be "The VDC concept..."

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
-In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall or a farm of firewalls and/or Web Application Firewalls (WAFs).
+[**Network virtual appliances**][NVA]. In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall instance or a farm of firewalls or web application firewall (WAF).
 
-Different LOBs commonly use many web applications, and these applications tend to suffer from various vulnerabilities and 
```
"...a VDC" should be "...the VDC"

TrishaMc replied:
This paragraph was deleted per your next comment. I searched and replaced "a VDC" with "the VDC." Are there any instances where it should stay "a VDC"?

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
-In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall or a farm of firewalls and/or Web Application Firewalls (WAFs).
+[**Network virtual appliances**][NVA]. In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall instance or a farm of firewalls or web application firewall (WAF).
 
-Different LOBs commonly use many web applications, and these applications tend to suffer from various vulnerabilities and 
```
You can strike this paragraph - it's redundant, and the real content begins in the next paragraph...

TrishaMc replied:
Deleted this paragraph.

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
-In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall or a farm of firewalls and/or Web Application Firewalls (WAFs).
+[**Network virtual appliances**][NVA]. In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall instance or a farm of firewalls or web application firewall (WAF).
 
-Different LOBs commonly use many web applications, and these applications tend to suffer from various vulnerabilities and 
```
"However, a single VDC implementation is required to be hosted within a single region because the subscription requirements do not allow spanning of regions. This makes a single VDC implementation vulnerable to regional outages."

TrishaMc replied:
Replaced the deleted paragraph with this new paragraph.

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
-In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall or a farm of firewalls and/or Web Application Firewalls (WAFs).
+[**Network virtual appliances**][NVA]. In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall instance or a farm of firewalls or web application firewall (WAF).
 
-Different LOBs commonly use many web applications, and these applications tend to suffer from various vulnerabilities and 
```
"...the VDCs and the distance between the VDC and..." should be "the VDC implementations and the distance between the VDC implementations..."

TrishaMc replied:
I found this phrase and made the change. However, it was in the paragraph that begins, "Azure datacenters are present in many regions worldwide."

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
-In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall or a farm of firewalls and/or Web Application Firewalls (WAFs).
+[**Network virtual appliances**][NVA]. In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall instance or a farm of firewalls or web application firewall (WAF).
 
-Different LOBs commonly use many web applications, and these applications tend to suffer from various vulnerabilities and 
```
"VDC" should be "VDC implementation" in each use here

TrishaMc replied:
You asked me to delete the paragraph shown here. I didn't want to search and replace all instances of VDC with "VDC implementation" in case that's wrong. Can you indicate which instances of "VDC" need to be replaced?

Petertaylor9999 commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
-In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall or a farm of firewalls and/or Web Application Firewalls (WAFs).
+[**Network virtual appliances**][NVA]. In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall instance or a farm of firewalls or web application firewall (WAF).
 
-Different LOBs commonly use many web applications, and these applications tend to suffer from various vulnerabilities and 
```
VDCs should be VDC implementations

TrishaMc replied:
The paragraph attached to this comment was deleted. I changed all instances of "VDCs" to "VDC implementations." Can you let me know if that's not correct?

Petertaylor9999 replied:
Sorry - yes, the paragraph should be deleted.

TrishaMc commented:
Hi @Petertaylor9999 and @BrianBlanchard: I'll make these changes.
Thanks,
Trisha

TrishaMc commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+A VDC isn't just the application workloads in the cloud. It's also the network, security, management, and infrastructure. Examples are DNS and directory services. It usually provides a private connection back to an on-premises network or datacenter. As more and more workloads move to Azure, it's important to think about the supporting infrastructure and objects that these workloads are placed in. ...
 
-A Virtual Data Center is essentially a collection of separate but related entities with common supporting functions, features, and infrastructure. By viewing your workloads as an integrated VDC, you can realize reduced cost due to economies of scale, optimized security through component and data flow centralization, along with easier operations, management, and compliance audits.
+A virtual datacenter is a collection of separate but related entities with common supporting functions, features, and infrastructure. By viewing your workloads as an integrated VDC, you can reduce costs from economies of scale and optimized security through component and data flow centralization. You also get easier operations, management, and compliance audits.
```
I made the same change on line 31: "The VDC isn't just the application workloads in the cloud." Is that correct?

Petertaylor9999 replied:
Yes.

TrishaMc commented on docs/vdc/networking-virtual-datacenter.md:
```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
 
 #### Security infrastructure
-Security infrastructure, in the context of a VDC, is mainly related to the segregation of traffic in the VDC's specific virtual network segment, and how to control ingress and egress flows throughout the VDC. Azure is based on multi-tenant architecture that prevents unauthorized and unintentional traffic between deployments, using Virtual Network (VNet) isolation, access control lists (ACLs), loa ...
+In the context of a VDC, security infrastructure mainly relates to traffic segregation in the VDC's specific virtual network segment and how to control ingress and egress flows throughout the VDC. Azure is based on multi-tenant architecture that prevents unauthorized and unintentional traffic between deployments. It uses virtual network isolation, access control lists (ACLs), load balancers, IP f ...
```
Acrolinx flags "This" by itself, so I changed this sentence to "This security infrastructure refers to ..."

VSC-Service-Account commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/vdc/networking-virtual-datacenter.md](https://github.com/TrishaMc/architecture-center/blob/networking-virtual-datacenter/docs/vdc/networking-virtual-datacenter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/vdc/networking-virtual-datacenter?branch=pr-en-us-856) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C27%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201811270049587727-856%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=FSI5rlEcIYzx3lGGwOwVXy%2FfMmMDIwnT3iUbX9W9DvU%3D&st=2018-11-27T00%3A49%3A08Z&se=2018-12-28T00%3A54%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

TrishaMc commented:
Revisions in progress.

VSC-Service-Account commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/vdc/networking-virtual-datacenter.md](https://github.com/TrishaMc/architecture-center/blob/networking-virtual-datacenter/docs/vdc/networking-virtual-datacenter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/vdc/networking-virtual-datacenter?branch=pr-en-us-856) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C27%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201811271952558185-856%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=7gDZywxXIYPzKpTef9WAOmDsBwbAzcuezLoLKYVmMuA%3D&st=2018-11-27T19%3A49%3A00Z&se=2018-12-28T19%3A54%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

TrishaMc commented:
Hi @Petertaylor9999 and @BrianBlanchard: I've made the changes from your comments. Please note:
- I used “enterprise” instead of “Enterprise” per the Microsoft style guide.
- I started making you changes by using the comments shown on this Conversation page. However, it's not accurate. It mistakenly shows the same paragraph for many of the comments. But based on that, I searched and replaced all instances of "a VDC" with "the VDC" and "VDCs" with "VDC implementations" because I couldn't tell from the comments on this page where to replace that. Then I looked on the dif page, and that page shows correctly. I went back through and tried to fix everything as it should be. However, there might be some instances where I replaced "a VDC" or "VDCs" when it shouldn't have been replaced. 
Can you let me know if I need to make any more changes to this article?
Thanks,
Trisha
